server/src/core/app.ts

```diff
@@ -2,12 +2,14 @@ import { UserService } from './user';
 import { TransactionService } from './payment';
 import { InvitationService } from './invitation';
 import { DonationDistributionService } from './distribution';
+import { StatService } from './stat';
```

Comment:
Call it `StatsService` not `StatService` (since statistics in these case is plural).